test/A2CGAE.jl

```diff
@@ -0,0 +1,48 @@
+@testset "A2CGAE" begin
+N_ENV = 16
+UPDATE_FREQ = 10
+env = MultiThreadEnv([CartPoleEnv(;T=Float32, seed=i) for i in 1:N_ENV])
+ns, na = length(rand(get_observation_space(env[1]))), length(get_action_space(env[1]))
+reset!(env, is_force=true)
+agent = Agent(
+    policy = QBasedPolicy(
+        learner=A2CGAELearner(
+            approximator=NeuralNetworkApproximator(
+                model=ActorCritic(
+                    actor=Chain(
+                        Dense(ns, 256, relu;initW = seed_glorot_uniform(seed = 17)),
+                        Dense(256, na;initW = seed_glorot_uniform(seed = 23)),
+                        softmax
+                    ),
+                    critic=Chain(
+                        Dense(ns, 256, relu;initW = seed_glorot_uniform(seed = 29)),
+                        Dense(256, 1;initW = seed_glorot_uniform(seed = 29)),
+                    )
+                ) |> gpu,
+                optimizer=ADAM(1e-3),
+                kind=HYBRID_APPROXIMATOR
+            ),
+            γ=0.99f0,
+            λ=0.99f0,
+            actor_loss_weight = 1.0f0,
+            critic_loss_weight = 0.5f0,
+            entropy_loss_weight = - 0.001f0
```

Comment:
It should be a positive number for the `entropy_loss_weight`. Sorry that it's my fault. I've corrected it in the test case of `A2C.jl`.